collector/cfappscollector.go

```diff
@@ -38,14 +38,66 @@ func (c *CfAppsCollector) GetAnnotations() []string {
 	return annotations
 }
 
+// getNewIbmcloudCfClient returns a new ibmcloud cf client
+func (c *CfAppsCollector) getNewIbmcloudCfClient(homeDir string) (*cfclient.Client, error) {
```

Comment:
Let's just call it getCfClient and have all the possible home directories tried here. Probably gave a loop that iterates over a list of home directories

Reply:
Done